tests/jobs/test_backfill_job.py

```diff
@@ -581,11 +581,41 @@ def test_backfill_run_rescheduled(self, dag_maker):
         ti.refresh_from_db()
         assert ti.state == State.SUCCESS
 
+    def test_backfill_skip_active_scheduled_dagrun(self, dag_maker):
+        dag = self._get_dummy_dag(
+            dag_maker,
+            dag_id="test_backfill_skip_active_scheduled_dagrun",
+            task_id="test_backfill_skip_active_scheduled_dagrun-1",
+        )
+        dag_maker.create_dagrun(
+            run_type=DagRunType.SCHEDULED,
+            state=State.RUNNING,
+        )
+
+        executor = MockExecutor()
+
+        job = BackfillJob(
+            dag=dag,
+            executor=executor,
+            start_date=DEFAULT_DATE,
+            end_date=DEFAULT_DATE + datetime.timedelta(days=2),
+        )
+        with patch.object(job.log, 'error') as mock_error:
+            job.run()
+            mock_error.assert_called()
```

Comment:
NIT: I'd also do if assert "Backfill cannot be created" in the error message. It just prevents the case that there is **another** error generated during the test.